aemedge/blocks/hero/hero.js

```diff
@@ -61,17 +60,27 @@ async function replacePlaceholderText(elem) {
  * @param {Element} block The footer block element
  */
 export default async function decorate(block) {
-  const isMediaBlend = block.classList.contains('media-blend') || getMetadata('template') === 'article';
+  const isArticle = getMetadata('template') === 'article';
+  const isMediaBlend = block.classList.contains('media-blend') || isArticle;
+  const placeholder = await fetchPlaceholders();
+
+  // extract block content
   const hero = document.createElement('udex-hero-banner');
-  const intro = block.querySelector('h6');
   const heading = block.querySelector('h1');
+  const eyebrow = block.querySelector('h6');
+  let eyebrowText = eyebrow?.textContent;
+  if (!eyebrowText && isArticle) { // if no eyebrow text is set, use the content type for articles
+    const placeholderText = placeholder[toCamelCase(`content-type/${getMetadata('content-type')}`)];
+    eyebrowText = placeholderText || getMetadata('content-type');
```

Comment:
`getMetadata('content-type')` must be converted to camel case e.g. **original-research**

Reply:
In this case, if we use it as fallback if no placeholder exists not here the content type value looks better.

Reply:
Yes thats fine. What I was wondering if we want to show the fallback value as **original-research** or **Original Research**?

Reply:
It will be original-research in capital letters. But given we have placeholders for all it should not show that at all.

Reply:
Ok I thought better would be 
`eyebrowText = placeholderText || toCamelCase(getMetadata('content-type'));`

Reply:
Ok, I did some more test locally, I agree, change it.